internal/gatewayapi/testdata/httproute-attaching-to-listener-with-backend-backendref-mixed-address-type.out.yaml

```diff
@@ -151,8 +151,9 @@ httpRoutes:
         status: "True"
         type: Accepted
       - lastTransitionTime: null
-        message: Unix domain sockets are not supported in backend references
-        reason: UnsupportedRefAddressFound
+        message: 'Failed to process route rule 0 backendRef 0: unix domain sockets
+          are not supported in backend references.'
+        reason: InvalidAddressType
```

Comment:
The previous reason `UnsupportedRefAddressFound` is not a standard Gateway API reason either.  I can change this back if prefered.